pyproject.toml

```diff
@@ -34,6 +34,68 @@ profile = "black"
 combine_as_imports = true
 ensure_newline_before_comments = false
 
+# -----------------------------------------------------------------------------
+#                            COVERAGE CONFIGURATION
+#    https://coverage.readthedocs.io/en/latest/config.html#run-command-line
+# -----------------------------------------------------------------------------
+#          Section configuring the collection of the coverage data:
+# -----------------------------------------------------------------------------
+[tool.coverage.run]
+# -----------------------------------------------------------------------------
+# The default is to run the test suite with the command "coverage run -m pytest"
+#
+command_line = "-m pytest"
+
+# -----------------------------------------------------------------------------
+# The source option specifies the directories to be measured:
+#
+source = ["scripts", "ocaml"]
+
+# -----------------------------------------------------------------------------
+# The omit option specifies files or directories to be omitted from the report:
+#
+omit = [  # These need some simple syntax update to be parseable:
```

Comment:
what is going on with these scripts? I understand that this PR is more for infra change, do we have another PR that does the syntax updates for these?